runner/agentd.py

```
@@ -0,0 +1,465 @@
+            except Exception as error:  # noqa: BLE001 - 予期しない失敗もクライアントに 500 で返す
+                agentd.audit({"verb": verb, "status": 500, "detail": repr(error)})
+                self.respond(500, {"error": "内部エラー"})
+                return
+            agentd.audit({"verb": verb, "status": 200, "detail": result})
+            self.respond(200, {"ok": True, **result})
+
+        def read_body(self):
+            try:
+                length = int(self.headers.get("Content-Length", "0"))
+            except ValueError:
+                raise RequestError(400, "Content-Length が不正")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)